### PR TITLE
card collapsing fixed in course and faq card

### LINKED ISSUE #80: The course view is not collapsing on clicking collapse button
ezgif com-video-to-gif

### PR DESCRIPTION
**Pull request by ArunTeltia**
#80 fixed , now the card is collapsing on clicking the "^" button

### CODE REVIEW

**Comment by ArunTeltia** [comment]
How to pass header rules and pages changed @Abhishek-kumar09

File: src/views/pages/Courses/Schedule.js [hunk 1]
```diff
@@ -194,16 +194,16 @@ function BatchDropBox({ course, batch, batchIndex, expanded, setExpanded }) {
...
                 }}
               >
                 <ExpandLessIcon />
               </IconButton>
             ) : (
-              <IconButton>
-                <ExpandMoreIcon />
-              </IconButton>
-            )}
+                <IconButton>
```
Comment (Abhishek-kumar09) [inline src/views/pages/Courses/Schedule.js:203]: please restore these indentations.

**Comment by Abhishek-kumar09** [comment]
> How to pass header rules and pages changed @Abhishek-kumar09

Are you asking about the page change and appbar change,etc, so you can get those in `src/Routes.js` file. Please tell me if you didn't get your answer.

**Review by Abhishek-kumar09 — APPROVED** [review]

File: src/views/pages/Courses/Schedule.js (hunk 1 above)
Comment (ArunTeltia) [reply src/views/pages/Courses/Schedule.js:203]: Okay right away

**Comment by ArunTeltia** [comment]
No i was talking about the all the checks via netlify as you see 2 cases are neutral and not tick mark

**Comment by Abhishek-kumar09** [comment]
> No i was talking about the all the checks via netlify as you see 2 cases are neutral and not tick mark

You don't have to worry about it currently, They are normal and good untill they don't show failed :)

**Comment by ArunTeltia** [comment]
I used the ``` npm run format ``` for formatting the code

**Review by KeenWarrior — APPROVED** [review]

**Pull request merged**